experiments/defaults.py

```diff
@@ -240,9 +249,10 @@ def default_train(
         tags: Any additional tags to add to the Wandb tracker.
         use_default_validation: Whether to use the default validation sets (currently Paloma).
         eval_harness_tasks: List of evaluation harness tasks. Defaults to the CORE set of tasks. Use () or [] to disable
+        is_speedrun: Whether this is a speedrun. If so, load appropriate val set and in the future, other metrics and settings we need for speedrun.
```

Comment:
Rather than adding a new arg to default train, let's just override the validation sets in default_speedrun?